com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-basic-unlit-structure.md

```diff
@@ -0,0 +1,177 @@
+# URP unlit basic shader
+
+This example shows a basic URP-compatible shader. This shader fills the mesh shape with a color predefined in the shader code.
```

Comment:
"... -compatible Unity shader." (This code creates a "Unity shader", not just a shader - a Unity shader is a data structure that includes but isn't limited to shader programs)